### ISSUE TITLE
Incoming Call FullScreen Intent On A Cordova Activity

### ISSUE THREAD

**Issue by KyleKernanRepo** [apache/cordova-android#1689]
# Bug Report

## Information
We currently have an app that receives an incoming call notification by receiving a full-screen intent. When the intent comes through we are passing the app as an activity to load. When the app has been loaded already by the user, the full-screen intent works as intended and we receive the incoming call. If the app is not loaded by the user or the system restarts the service in the background the UI does not display correctly.  The sound notification comes through and we can hear the ringing, but the phone does not wake up or display the incoming call UI screen. We are passing a Cordova activity to the fullscreen intent, but it only works correctly when the app is loaded previously by the user.  The debug logs show that JS is loading but the UI doesn’t actually load on the phone during the lock screen, the phone needs to be unlocked manually in order for the UI to display.  

### Version information
Device: Samsung S23 Running Android 13
Cordova Version: cordova-lib@12.0.1
IDE: Android Studio

## Checklist

- [ x ] I searched for existing GitHub issues
- [ x ] I updated all Cordova tooling to most recent version
- [ x ] I included all the necessary information above

**Comment by parveshneedhoo** [comment]
@KyleKernanRepo I'm experiencing the same issue. Have you found a solution for it or a workaround (besides opening the app beforehand)?

**Comment by KyleKernanRepo** [comment]
@parveshneedhoo Yes the solution I found was you need to set your flags within the onCreate() as well as onNewIntent().

getWindow().addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON|
            WindowManager.LayoutParams.FLAG_DISMISS_KEYGUARD|
            WindowManager.LayoutParams.FLAG_SHOW_WHEN_LOCKED|
            WindowManager.LayoutParams.FLAG_TURN_SCREEN_ON);
